shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -139,10 +140,18 @@
 
   NSObject<FlutterPlatformViewFactory>* factory = factories_[viewType].get();
   if (factory == nil) {
-    result([FlutterError errorWithCode:@"unregistered_view_type"
-                               message:@"trying to create a view with an unregistered type"
-                               details:[NSString stringWithFormat:@"unregistered view type: '%@'",
-                                                                  args[@"viewType"]]]);
+    result([FlutterError
+        errorWithCode:@"unregistered_view_type"
+              message:[NSString stringWithFormat:@"A UIKitView widget is trying to create a "
+                                                 @"PlatformView with an unregistered type: < %@ >",
+                                                 viewTypeString]
+              details:@"If you are the author of the PlatformView, make sure `registerViewFactory`"
```

Comment:
Can you double check the spacing on these?
```suggestion
              details:@"If you are the author of the PlatformView, make sure `registerViewFactory` "
```

Reply:
Done!